dask/dataframe/tseries/tests/test_resample.py

```diff
@@ -8,6 +9,14 @@
 import dask.dataframe as dd
 
 
+if LooseVersion(pd.__version__) >= '0.18.0':
```

Comment:
Maybe we can put the logic here.

https://github.com/dask/dask/blob/master/dask/dataframe/utils.py#L100